modules/metastore/src/Reference/ReferenceLookup.php

```diff
@@ -64,14 +64,31 @@ public function getReferencers(string $schemaId, string $referenceId, string $pr
       [$identifier, $metadata] = $this->decodeJsonMetadata($item);
       $propertyValue = $metadata->{$propertyId};
       // Check if uuid is found either directly or in an array.
-      $idIsValue = $referenceId == $propertyValue;
-      $idInArray = is_array($propertyValue) && in_array($referenceId, $propertyValue);
-      if ($idIsValue || $idInArray) {
-        $referencers[] = $identifier;
-      }
+      $idIsValue = is_string($propertyValue) && str_starts_with($propertyValue, $referenceId);
```

Comment:
Were we going to explicitly add the polyfill library to support `str_starts_with()`?